docs/design/logrotate.md

```diff
@@ -0,0 +1,73 @@
+# Ceph CSI Log Rotate Design Document
+
+Log Rotate is the ability to rotate the logs by controlling the size of log files.
+With csi it will be includes to all the csi pods which includes csi daemons and provisoners.
+Using logrotate will create a log file at the hostpath and enable the logrotator sidecar container.
+Logrotator sidecar container will make sure that log files are rotated as per the inputs.
+
+Logroate configuration, 
+
+`CephCSIOperatorConfig CRD`:
+
+```yaml
+kind: CephCSIOperatorConfig 
+apiVersion: csi.ceph.io/v1alpha1
+….
+spec: 
+    logLevel: 1 
+    driverSpecDefaults: 
+        log:
+            logLevel: 5
+            logRotator:
+                # one of: hourly, daily, weekly
+                Periodicity: daily
+                # SUFFIX may be 'M' or 'G' 
+                MaxLogSize: 500M 
+                MaxFiles: 5
+                logHostPath: /var/lib/log 
```

Comment:
`/var/lib/log` does not seem like a valid example. In the rook scenario, this would be set to `/var/lib/rook`, right? What path will be the default if not specified? Let's set this example to that default path.
```suggestion
                logHostPath: /var/lib/cephcsi
```

Reply:
DOne